test/KeyRegistry/KeyRegistry.symbolic.t.sol

```diff
@@ -10,6 +10,7 @@ import {KeyRegistryHarness} from "./utils/KeyRegistryHarness.sol";
 import {IdRegistry} from "../../src/IdRegistry.sol";
 import {StubValidator} from "../Utils.sol";
 
+/// @custom:halmos --default-bytes-lengths 0,32,1024,65
```

Comment:
the default size candidates for bytes/string are 0, 65, and 1024. here we also include 32 to account for the key size.